setup.py

```diff
@@ -187,6 +187,9 @@ def write_version(filename=os.path.join(*['airflow',
 segment = ['analytics-python>=1.2.9']
 sendgrid = ['sendgrid>=5.2.0']
 slack = ['slackclient>=1.0.0']
+kubernetes = ['kubernetes>=3.0.0',
```

Comment:
Why is this duplicated here? The `kubernetes` dependencies are already in the file

Reply:
Hmm not sure - definitely not related to what I was working on. I will remove, rebase and push.